auto_submit/lib/validations/ci_successful.dart

```diff
@@ -140,17 +140,10 @@ class CiSuccessful extends Validation {
     final String overrideTreeStatusLabel = config.overrideTreeStatusLabel;
     log.info('Validating name: ${slug.name}/$prNumber, statuses: $statuses');
 
-    final List<ContextNode> staleStatuses = <ContextNode>[];
     for (ContextNode status in statuses) {
       // How can name be null but presumed to not be null below when added to failure?
       final String? name = status.context;
 
-      // If the account author is a roller account do not block merge on flutter-gold check.
-      if (isToEngineRoller(author, slug) && name == 'flutter-gold') {
```

Comment:
I think this is the only code that needs to be removed here to avoid skipping and enforcing the failure.

Reply:
Ah thinks Ricardo, this is my bad - it looks like I had an older branch. Let me redo this fix and tag you.